std_msgs/Header header

uint32 id
float distance
uint32 brief_stop
